mono/metadata/reflection.c

```diff
@@ -8623,7 +8633,10 @@ create_custom_attr (MonoImage *image, MonoMethod *method, const guchar *data, gu
 	named = p;
 	attr = mono_object_new (mono_domain_get (), method->klass);
 
-	mono_runtime_invoke (method, attr, params, &exc);
+	mono_runtime_invoke_checked (method, attr, params, &exc, error);
+	if (!mono_error_ok (error))
+		return NULL;
+
```

Comment:
This leaks `params` if there's an error.  (The old code leaked too if `mono_runtime_invoke` raised an exception) (see `goto fail`, below)